### PR TITLE
Usage reports cloud docs.

### PR DESCRIPTION
**Pull request by legander**
Probably needs typo/swenglish checking.

TODO
- [x] Copy about how to use the UI and what information is being displayed.
- [x] Link "Read how to add thresholds to your tests." to a thresholds docs. 
- [x] Need to check with @Griatch about "How do I know which tests add to the fail rate statistic?" and "All members that have either **created** or **started tests** in the project"
- [x] Add copy for what value the UI brings.
- [x] Check with @ppcano about the copy or design now when there is one list item short 🙀

Screenshot 2021-01-05 at 16 03 56

### CODE REVIEW

**Comment by ppcano** [comment]
I think `one list item` is fine. It will be fixed when we release PDF reports.

I would rephrase `Project performance with Usage reports` with something like 
See team and testing activity with [usage reports]().


Feature value proposal: the usage report helps you keep track of everything happening in your team’s project and quickly follow the activity of your performance testing projects.

**Comment by legander** [comment]
Will start preparing a release of the app with the updated usage-reporting UI.

**Review by ppcano — APPROVED** [review]
I will merge this when the feature is available.

**Comment by legander** [comment]
It is available, but currently not visible to all users. Lets coordinate a release tomorrow 👍🏻

**Comment by sniku** [comment]
I think it's a bit confusing that we called the K6 OSS analytics "Usage reports"[0] and this feature is also "Usage reports". 
This is bound to confuse some people, not to mention Google :smile:  Maybe we should rename one of these features before the release? 

[0] [k6 OSS Usage reports](https://mdr-ci.staging.k6.io/docs/refs/pull/183/merge/misc/usage-reports)
[1] [k6 cloud Usage reports](https://mdr-ci.staging.k6.io/docs/refs/pull/183/merge/cloud/project-and-team-management/usage-reports)

**Comment by ppcano** [comment]
> I think it's a bit confusing that we called the K6 OSS analytics "Usage reports"[0] and this feature is also "Usage reports"[1].

Fixed by https://github.com/loadimpact/k6-docs/commit/f67ad74b4e0b707d9d2038d9755d4090d271f033

**Comment by legander** [comment]
Thanks @ppcano 🤙 
Found a regression in the app (unrelated to this feature though) but want to do some thorough testing before sending out a release. Will probably do it later today or tmrw morning. Will let you know when it time to merge this PR!
